docs/quickstart-eks.md

```diff
@@ -0,0 +1,264 @@
+
+If you already have a working User Pool in AWS Cognito that you would like to use for authenticating users, you will need to retrieve its ID in the form `${AWS_REGION}-${USER_POOL_ID}`, and write it down. Using this `Pool Id`, skip at the end of this section.
+
+Else, If you are new to using BKPR on EKS, or if you want to create a new User Pool in AWS Cognito, please follow these steps:
+
+1. Browse to the [Cognito](https://console.aws.amazon.com/cognito/) module in the AWS Console
+2. Click on the `Manage User Pools` button
+3. Click on the `Create a user pool` button on the top-right corner
+4. Enter a valid `Pool name` like `eks-test`
+5. Click on the `Step through settings` button
+6. In the `Attributes` section leave all the checkboxes unticked as shown below:
+
+<p align="center"><img src="eks/1-Attributes.png" width=840/></p>
+
+7. Click on the `Next step` button
+8. In the `Policies` section leave all the checkboxes unticked and select `Only allow administrators to create users` as shown below:
+
+<p align="center"><img src="eks/2-Policies.png" width=840/></p>
+
+9. Click on the `Next step` button
+10. In the `MFA and verifications` section select `No verification` as shown below:
+
+<p align="center"><img src="eks/3-MFA.png" width=840/></p>
+
+11. Click on the `Next step` button
+12. In the `Message customizations` section do not change anything and click on the `Next step` button
+13. In the `Tags` section you can add additional tags for your EKS cluster, if needed
+14. Click on the `Next step` button
+15. In the `Devices` section do not change anything and click on the `Next step` button
+16. In the `App clients` section do not change anything and click on the `Next step` button
+17. In the `Triggers` section do not change anything and click on the `Next step` button
+18. In the `Review` section you will see a quick summary of the user pool that will be created, like the onw shown below:
+
+<p align="center"><img src="eks/4-Review.png" width=840/></p>
+
+19. Click on the `Create pool` button to create the user pool
+20. The user pool will be created and the `General settings` page will be displayed, where you can find the User Pool ID and ARN:
+
+<p align="center"><img src="eks/5-User-Pool.png" width=840/></p>
+
+Take note of the `Pool Id` and export its value:
+
+  ```bash
+  export AWS_COGNITO_USER_POOL_ID=eu-central-1_sHSdWT6VL
+  ```
+
+In order to access protected resources which require authentication like Prometheus, Kibana or Grafana, you will need to configure users in the newly created user pool. The next steps highlight how to create a test user which can be used to access these protected resources:
+
+1. Browse to the [Cognito](https://console.aws.amazon.com/cognito/) module in the AWS Console
+2. Click on the `Manage User Pools` button
+3. Click on the button for the user pool you just created
+4. Click on the `Users and groups` option from the menu on the left
+5. Click on the `Create user` button
```

Comment:
I would suggest changing this entire sequence to:

2. Navigate to **Manage User Pools > _YOUR_USER_POOL_ > Users and Groups > Create user**